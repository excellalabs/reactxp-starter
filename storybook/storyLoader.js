
// Auto-generated file created by react-native-storybook-loader
// Do not edit.
//
// https://github.com/elderfo/react-native-storybook-loader.git

function loadStories() {
  require('../src/components/Display/Display.story.tsx');
  require('../src/containers/Hello/Hello.story.tsx');
  
}

const stories = [
  '../src/components/Display/Display.story.tsx',
  '../src/containers/Hello/Hello.story.tsx',
  
];

module.exports = {
  loadStories,
  stories,
};
