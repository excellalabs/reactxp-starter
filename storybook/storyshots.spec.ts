
import { loadStories } from './storyLoader'
import initStoryshots from '@storybook/addon-storyshots'

initStoryshots()
