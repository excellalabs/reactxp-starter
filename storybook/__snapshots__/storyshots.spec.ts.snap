// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Display main display 1`] = `
<RX.View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <RX.Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Your App
  </RX.Text>
  <RX.Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  />
</RX.View>
`;

exports[`Storyshots Hello shows name "Joseph" 1`] = `
<RX.View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <RX.Text
    style={
      Object {
        "color": "#110022",
        "fontSize": 18,
        "paddingBottom": 10,
        "textAlign": "center",
      }
    }
  >
    Hello, 
    Joseph
  </RX.Text>
  <RX.Button
    onPress={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#330066",
          "borderRadius": 5,
          "justifyContent": "center",
          "paddingBottom": 10,
          "paddingLeft": 20,
          "paddingRight": 20,
          "paddingTop": 10,
        },
      ]
    }
  >
    <RX.Text
      style={
        Array [
          Object {
            "color": "#FFFFFF",
            "fontSize": 20,
            "textAlign": "center",
          },
        ]
      }
    >
      Random Name
    </RX.Text>
  </RX.Button>
</RX.View>
`;
